README.md

```diff
@@ -237,7 +92,7 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/hakichris/Awesome-Books-Project/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] The link to your MIT is not correct. It should be `./MIT.md` and not `./LICENSE.md`. Also, in your `MIT.md` file, kindly change the phrase _**[YOUR NAME]**_ to your actual name and all references to the phrase _**[APP TYPE]**_ to the name of your project. 👍

Reply:
@Gambit142 Resolved, Thank you